docs/packages/dara-components/plotting/introduction.mdx

```diff
@@ -17,7 +17,7 @@ In this page we will cover how to add the extension to your project and a quick
 
 ## Getting Started
 
-### Add your first `dara.components.plotting.bokeh.Bokeh` Component
+### Add your first `dara.components.plotting.bokeh.bokeh.Bokeh` Component
```

Comment:
Does class linking work in the title? My experience was that it does not and that docusaurus gets a bit confused with `bokeh.bokeh` when it is in the title :( you can try to move the class linking bit in the text and leave the title as just text if it does not work :)